Zend/tests/final_properties/inheritance/inheritance_success2.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test that non-final properties can be overwritten by final properties
+--FILE--
+<?php
+
+class Foo
+{
+    protected static int $property1;
+    protected int $property2;
+}
+
+class Bar extends Foo
+{
+    final protected static int $property1;
+    final protected int $property2;
```

Comment:
This looks unsound. Foo may be assuming that it can write to the properties.

Reply:
Oh, you are right of course.